roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/main.yml

```diff
@@ -61,22 +61,22 @@
   when: __sap_hypervisor_node_preconfigure_register_worker_memory_gib | int >= 512
 
 - name: Include prepare
-  ansible.builtin.include_tasks: prepare.yml
+  ansible.builtin.include_tasks: "platform/{{ sap_hypervisor_node_platform }}/prepare.yml"
```

Comment:
I see that folders respective to possible platforms do not have all the task files.
I think this is just an oversight. 
Maybe worth adding content to that folders to this PR.

Reply:
The switch which platform to load is one level up in here `roles/sap_hypervisor_node_preconfigure/tasks/main.yml`. So I see no reason why all files would have to be available for other platforms there.